bounties/level-1/basic-account-viewer.md

```diff
@@ -0,0 +1,45 @@
+# Basic account viewer
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 200 XLM | 1 Credits | Continuos
+
+## Context
+https://github.com/tyvdh/stellar-quest-bounties/issues?q=basic-account-viewer.md
+
+## Description
+
+### What is this task?
+
+When building websites ontop of Stellar, there needs to be a way to access a user's Stellar account. This is where web-wallets like [Albedo](https://albedo.link/), [Rabet](https://rabet.io/) and [Freighter](https://www.freighter.app/) come in. They allow websites to easily and securely acces your Stellar account.
+
+Your task is to build a simple website which allows you to sign in using a web-wallet like Albedo, Freighter or Rabet.
+
+The website should show some basic information about the account currently logged in:
+ * The asset-codes and balances of the account's assets should be shown.
```

Comment:
Would you consider just showing basic information as asset-code and balance enough, as additional information is available from the well-know file?